src/app/shared/utils/array.utils.ts

```diff
@@ -50,3 +50,9 @@ export function getArrayDifference<T>(bigArray: T[], smallArray: T[]): T[] {
 
   return bigArray.filter(item => !smallArray.includes(item));
 }
+
+export function arrayIntersection(array1: string[], array2: string[]): string[] {
```

Comment:
This method should be made generic in order to be reusable. Look at the method a few lines above.

Reply:
Oh yes :)